src/test/java/seedu/address/logic/commands/DelEventCommandTest.java

```diff
@@ -71,31 +71,31 @@ public void execute_invalidIndexFilteredList_throwsCommandException() {
         // ensures that outOfBoundIndex is still in bounds of address book list
         assertTrue(outOfBoundIndex.getZeroBased() < model.getAddressBook().getEventList().size());
 
-        DelEventCommand deleventCommand = new DelEventCommand(outOfBoundIndex);
+        DelEventCommand delEventCommand = new DelEventCommand(outOfBoundIndex);
 
-        assertCommandFailure(deleventCommand, model, Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
+        assertCommandFailure(delEventCommand, model, Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
     }
 
     @Test
     public void equals() {
-        DelEventCommand deleteFirstCommand = new DelEventCommand(INDEX_FIRST_EVENT);
-        DelEventCommand deleteSecondCommand = new DelEventCommand(INDEX_SECOND_EVENT);
+        DelEventCommand delEventFirstCommand = new DelEventCommand(INDEX_FIRST_EVENT);
+        DelEventCommand delEventSecondCommand = new DelEventCommand(INDEX_SECOND_EVENT);
 
         // same object -> returns true
-        assertTrue(deleteFirstCommand.equals(deleteFirstCommand));
+        assertTrue(delEventFirstCommand.equals(delEventFirstCommand));
 
         // same values -> returns true
-        DelEventCommand deleteFirstCommandCopy = new DelEventCommand(INDEX_FIRST_EVENT);
-        assertTrue(deleteFirstCommand.equals(deleteFirstCommandCopy));
+        DelEventCommand delEventFirstCommandCopy = new DelEventCommand(INDEX_FIRST_EVENT);
+        assertTrue(delEventFirstCommand.equals(delEventFirstCommandCopy));
 
         // different types -> returns false
-        assertFalse(deleteFirstCommand.equals(1));
+        assertFalse(delEventFirstCommand.equals(1));
 
         // null -> returns false
-        assertFalse(deleteFirstCommand.equals(null));
+        assertFalse(delEventFirstCommand.equals(null));
 
         // different person -> returns false
```

Comment:
I think there is a typo in the comment. Should it be `different event commands -> returns false`?